src/App.jsx

```diff
@@ -14,33 +18,89 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD = {
+  alphabeticaly: 'alphabeticaly',
+  length: 'length',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function sortGoods(input, sortField, reverse) {
+  const preparedGoods = [...input];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD.alphabeticaly:
+          return good1.localeCompare(good2);
+        case SORT_FIELD.length:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (reverse === 'desc') {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reverse, setReverse] = useState('asc');
```

Comment:
The `reverse` state is initialized as a string ('asc'), but it might be more appropriate to use a boolean value for clarity and consistency. Consider changing the logic to use a boolean instead.